### PR TITLE
Les72

### PR DESCRIPTION
**Pull request by Dmitryshl**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -4,29 +4,24 @@
   mount Rswag::Ui::Engine => '/api-docs'
   mount Rswag::Api::Engine => '/api-docs'
   post '/sign_up', to: 'sign_up#create'
-  post '/login', to: 'sign_up#login', defaults: { format: :json }
```
Comment (fooooxmr) [inline config/routes.rb:7]: Why login endpoint is removed?

File: swagger/v1/swagger.yaml [hunk 2]
```diff
@@ -396,24 +387,22 @@ paths:
                   value:
                     status: 404
                     error: Not Found
-  "/{locale}/check_username?name={username}":
+  "/{locale}/check_email?email={email}":
```
Comment (fooooxmr) [inline swagger/v1/swagger.yaml:390]: You need to remove locale from the swagger also.

File: swagger/v1/swagger.yaml [hunk 3]
```diff
@@ -422,28 +411,29 @@ paths:
...
                 type: object
                 properties:
-                  username_exists:
+                  email_exists:
                     type: boolean
+                    example: 'true'
                 required:
-                  - username_exists
-  "/{locale}/check_email?email={email}":
+                - email_exists
```
Comment (fooooxmr) [inline swagger/v1/swagger.yaml:418]: Why we are renamed username_exists to email_exists?

**Review by fooooxmr — CHANGES_REQUESTED** [review]

**Review by evrusha — COMMENTED** [review]
I'll re-review after the conversations be resolved.

**Comment by Terapool** [comment]
See https://github.com/IT-Academy-Ruby/lessoner_project_backend/pull/59

**Pull request closed without merging**